src/lib/hooks/useIsDarkMode.ts

```diff
@@ -0,0 +1,7 @@
+import { useColorMode } from '@chakra-ui/react';
+
+export const useIsDarkMode = (): boolean => {
+  const { colorMode } = useColorMode();
+
+  return colorMode !== 'light';
+};
```

Comment:
nit++: Maybe maybe a hook called `useColorMode` which returns a `mode` function with the same functionality as the `mode` in theme tools? Then you can avoid a lot of the logic (basic but repeated many times) and keep consistent with theme `mode` usage.

Reply:
I will definitely document all this stuff once it goes through